Makefile

```diff
@@ -56,7 +56,10 @@ ENHANCEDDOCTARGET = enhanced-html
 ENHANCEDDOCSOURCE = util/enhanced-doc
 LATEXDIR = latex
 COQDOCLATEXOPTIONS := -latex -utf8 --body-only
-COQDEFS := --language=none -r '/^[[:space:]]*\(Local[[:space:]]+\)?\(Axiom\|Theorem\|Class\|Instance\|Let\|Ltac\|Definition\|Identity Coercion\|Lemma\|Record\|Remark\|Structure\|Fixpoint\|Fact\|Corollary\|Let\|Inductive\|Coinductive\|Notation\|Proposition\|Module[[:space:]]+Import\|Module\)[[:space:]]+\([[:alnum:]'\''_]+\)/\3/'
+COQDEFS := --language=none																																			       \
+	-r '/^[[:space:]]*\(Local[[:space:]]+\)?\(Axiom\|Theorem\|Class\|Instance\|Let\|Ltac\|Definition\|Identity Coercion\|Lemma\|Record\|Remark\|Structure\|Fixpoint\|Fact\|Corollary\|Let\|Inductive\|Coinductive\|Notation\|Proposition\|Module[[:space:]]+Import\|Module\)[[:space:]]+\([[:alnum:]'\''_]+\)/\3/' \
```

Comment:
One could split it up, but that would make it harder to detect other typos, those where one of the many
lines differs from another.  I recommend not doing it.
```
	-r '/^[[:space:]]*\(\(Local\|Global\)[[:space:]]+\)?Theorem[[:space:]]+\([[:alnum:]'\''_]+\)/\4/' \
	-r '/^[[:space:]]*\(\(Local\|Global\)[[:space:]]+\)?Lemma[[:space:]]+\([[:alnum:]'\''_]+\)/\4/' \
	...
```